perfdash/perfdash.go

```diff
@@ -45,18 +46,26 @@ var (
 	globalConfig = make(map[string]string)
 )
 
-func initGoogleDownloaderOptions() {
+// GCSDownloaderOptions is an options for GoogleGCS specific Downloader.
+type GCSDownloaderOptions struct {
```

Comment:
Do you need this object? As far as I can tell, it's not passed anywhere. I think you can just load this variable similar to on line 40.

Reply:
I removed GSCDownloaderOptions and initGoogleDownloaderOptions and loading these vars per your suggestion :)